README.md

```diff
@@ -334,16 +334,13 @@ Use `sfcc-ci --help` or just `sfcc-ci` to get started and see the full list of c
     user:create [options]                                           Create a new user
     user:update [options]                                           Update a user
     user:delete [options]                                           Delete a user
-    slas:tenant:list [options]                                      Lists all tenants that belong to a given organization
-    slas:tenant:add [options]                                       Adds a SLAS tenant to a given organization or updates an existing one
-    slas:tenant:get [options]                                       Gets a SLAS tenant from a given organization
-    slas:tenant:delete [options]                                    Deletes a SLAS tenant from a given organization
-    slas:client:add [options]                                       Adds a SLAS client to a given tenant or updates an existing one
-    slas:client:get [options]                                       Gets a SLAS client from a given tenant
-    slas:client:list [options]                                      Lists all SLAS clients that belong to a given tenant
-    slas:client:delete [options]                                    Deletes a SLAS client from a given tenant
-
-  Environment:
+    slas:tenant:add [options]                                       Add or update SLAS a tenant.
```

Comment:
As described in https://github.com/SalesforceCommerceCloud/sfcc-ci/pull/282#issuecomment-1040215448, I'd suggest to rename to `:create`, instead of `:add`, to follow existing patterns.